ajax/libs/svg.js/package.json

```diff
@@ -10,22 +10,19 @@
     "animation",
     "lightweight"
   ],
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/svgdotjs/svg.js.git",
-    "fileMap": [
-      {
-        "basePath": "dist",
-        "files": [
-          "*"
-        ]
-      }
-    ]
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/svgdotjs/svg.js/"
   },
+  "npmName": "@svgdotjs/svg.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*"
```

Comment:
Hi, @Fuzzyma I'm wondering `svg.node.js` is for node environment, is that so?

Reply:
Yes thats true. Shall I exclude it? If so, is `!(.*)node(.*)` sufficiant?

Reply:
Maybe `**/!(*.node.*)`? It grabs all files except for the ones with the filename`.node.` recursively.

Reply:
Alright, will use that. Thank you!